### DISCUSSION TITLE
AI in Components

### DISCUSSION THREAD

**Discussion by PWagner1** [Krypton-Suite/Standard-Toolkit#1378] [General]
Seems like Microsoft is going to introduce AI to .NET, this includes WinForms https://devblogs.microsoft.com/dotnet/introducing-dotnet-smart-components/

**Comment by Smurf-IV** [comment]
This will / maybe for V100....
Emphasis mine:

The .NET Smart Components are an experiment and are initially available for Blazor, MVC, and Razor Pages with .NET 6 and later. _We **expect** to provide components for other .NET UI frameworks as well, like .NET MAUI, WPF, and Windows Forms._

**Reply by PWagner1** [reply]
Yes, I'm guessing .NET 10/11
